CMakeLists.txt

```diff
@@ -426,7 +426,7 @@ endif ()
 
 if (APPLE AND NOT IOS)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=x86-64 -fvisibility=default -std=c++11")
-  if (NOT OpenSSL_DIR)
```

Comment:
Does this fetch and install OpenSSL automatically? That seems odd to have in the Monero build process.

Reply:
If `OPENSSL_ROOT_DIR` is not set manually it will check if `openssl` installed with brew and get its path. Nothing gets fetched or downloaded.